### PR TITLE
fix: create user

### PR DESCRIPTION
**Pull request by markusahlstrand**
We always created the user with email, but still validated what the client passed. This should fix the create user from auth-admin..

### CODE REVIEW

File: migrate/planetscale.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ const dialect = new PlanetScaleDialect({
     fetch(new Request(opts, { ...init, cache: undefined })),
 });
 
-// migrateToLatest(dialect)
-migrateDown(dialect)
+migrateToLatest(dialect)
```
Comment (markusahlstrand) [inline migrate/planetscale.ts:12]: Hmm.. though we fixed this in another PR?

File: src/routes/management-api/users.ts [hunk 2]
```diff
@@ -182,7 +182,7 @@ export class UsersMgmtController extends Controller {
       id: `email|${userIdGenerate()}`,
       name: user.name || email,
       provider: "email",
-      connection: "email",
+      connection: user.connection || "email",
```
Comment (markusahlstrand) [inline src/routes/management-api/users.ts:185]: Use the value passed in the body and default to email.

File: migrate/planetscale.ts (hunk 1 above)
Comment (danielgent) [reply migrate/planetscale.ts:12]: that's quite worrying!

https://github.com/sesamyab/auth/pull/540 this is the PR :smile: 

It was merged in :thinking:

Comment (danielgent) [reply migrate/planetscale.ts:12]: Yes this is on `main`!  see https://github.com/sesamyab/auth/blob/main/migrate/planetscale.ts#L12

This kind of thing shakes my faith in github :joy:

File: src/routes/management-api/users.ts (hunk 2 above)
Comment (danielgent) [reply src/routes/management-api/users.ts:185]: But we shouldn't do this! I did this for a reason :joy: 

Then we'll end up with lots of unlinked password accounts that our microservices are creating

Comment (markusahlstrand) [reply src/routes/management-api/users.ts:185]: It's still validating that you don't create a password account...

Comment (markusahlstrand) [reply src/routes/management-api/users.ts:185]: Currently we default to "email" but we still fail the check which isn't great.

Comment (danielgent) [reply src/routes/management-api/users.ts:185]: Right right, understood

**Pull request merged**